packages/gdbprofiler/gdbprofiler.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "copy@copy.sh"
+authors: "copy"
+homepage: "https://github.com/copy/gdbprofiler"
+bug-reports: "https://github.com/copy/gdbprofiler/issues"
+license: "LGPL-2.1 with OCaml linking exception"
+dev-repo: "https://github.com/copy/gdbprofiler.git"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  [make]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: ["ocamlfind" "remove" "gdbprofiler"]
+depends: [
+  "ocamlfind" {build}
+  "menhir"
+  ("extlib" | "extlib-compat")
+  "lwt" {>= "2.4.6"}
+  "ppx_deriving_yojson" {>= "2.0"}
+  "oasis" {build & >= "0.4"}
+  "containers"
```

Comment:
The CI Alpine 3.5 test selected containers.0.4 for some as-yet-undetermined reason, but it is a valid choice as there is no lower constraint here. If you force a more recent version (e.g. `>="0.22"`) here then the whole thing should pass and upgraded will be more reliable for users.